data/_ui/tooltips.txt

```diff
@@ -360,6 +360,24 @@ tip "jump mass cost:"
 tip "jump base mass:"
 	`Modifies the impact your ship's mass has on the jump mass cost. Ships below this mass will have a cheaper jump cost than the base jump fuel cost, while ships above this mass will have more expensive jumps.`
 
+tip "jumpdrive damage:"
+	`Damage received when using a Jump Drive. Damage is applied regardless of resistance.`
+
+tip "hyperdrive damage:"
+	`Damage received when using a Hyperdrive. Damage is applied regardless of resistance.`
+
+tip "jumpdrive shield damage:"
+	`Shield damage received when using a Jump Drive. Damage is applied regardless of resistance.`
```

Comment:
"regardless of ~~resistance~~ protection"  I think shield protection should still apply IMO

Reply:
changed it to protection, I do not agree on applying the protection values